.github/pull_request_template.md

```diff
@@ -17,7 +17,10 @@ See Story: [FSA22V2-ISSUE](https://sparkbox.atlassian.net/browse/FSA22V2-ISSUE)
 
 1. Make sure all PR Checks have passed (GitHub Actions, CircleCI, Code Climate, etc).
 2. Pull down all related branches.
-3. Confirm all tests pass.
+3. In terminal: npm run dev.
```

Comment:
This is personal preference, but I like to wrap commands in backticks so it's clear what is and isn't a command, like `npm run dev`.

Reply:
I like that too! Thank you :)